### PR TITLE
Add svg exports for cfgs

### PR DESCRIPTION
**Pull request by jeremy-rifkin**
Part 1 of implementing #4419 (cc @Saldivarcher)

There's a design decision to be made as far as themes go, should the export be theme-aware? At the moment it is and I've added a background rect to the svg to insert the pane's background color.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,9 +21,10 @@
   },
   "dependencies": {
     "@flatten-js/interval-tree": "^1.0.18",
-    "@fortawesome/fontawesome-free": "^5.15.4",
+    "@fortawesome/fontawesome-free": "^6.2.1",
```
Comment (jeremy-rifkin) [inline package.json:24]: Are we ok bumping the version here? I found I needed to bump for `file-arrow-down`, even though it was added in 5.1.0.

File: package.json [hunk 2]
```diff
@@ -21,9 +21,10 @@
...
     "@sentry/browser": "^6.16.1",
     "@sentry/node": "^6.16.1",
+    "@types/file-saver": "^2.0.5",
```
Comment (jeremy-rifkin) [inline package.json:27]: May as well grab the type declarations for file-saver

File: static/panes/cfg-view.ts [hunk 3]
```diff
@@ -165,7 +184,7 @@ export class Cfg extends Pane<CfgState> {
             const prevZoom = this.state.zoom;
             this.state.zoom += delta;
             if (this.state.zoom >= MINZOOM) {
-                this.graphElement.style.transform = `scale(${this.state.zoom})`;
+                this.zoom(this.state.zoom);
```
Comment (jeremy-rifkin) [inline static/panes/cfg-view.ts:187]: This will conflict with #4423, I'll fix the merge conflict when it happens

File: tsconfig.frontend.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     /* Code generation */
     "inlineSources": true,
     "strictPropertyInitialization": false,
-    "lib": ["dom", "es5"]
+    "lib": ["dom", "es5", "dom.iterable"]
```
Comment (jeremy-rifkin) [inline tsconfig.frontend.json:10]: Needed for `NodeList.entries()`/`NodeListOf<...>.entries()`, with this change I think some other code can be cleaned up too

File: package.json (hunk 1 above)
Comment (RubenRBS) [reply package.json:24]: Yes, I dont see any breaking changes for us in fa 6. They dropped support for ie but we have done so already so all's good

File: tsconfig.frontend.json (hunk 4 above)
Comment (RubenRBS) [reply tsconfig.frontend.json:10]: TIL why this is a separate lib, nice.

**Review by RubenRBS — APPROVED** [review]
:+1:

**Pull request merged**